index.md

```diff
@@ -0,0 +1,105 @@
+# Astropy mentoring program
+
+```{warning}
+The Astropy mentoring scheme is still being set up, and this
+documentation is not ready for use.
+```
+
+## Goals of the program
+
+While the Astropy project has been successful in attracting code contributions
+and feedback from the community, one of the biggest challenges for the
+sustainability of the project is training people to take on formal roles and
+responsibilities in the project, such as taking on maintainer roles for packages
+or sub-packages, or taking on other formal roles in the project.
```

Comment:
Please don't start with a negative. I recommend cutting this. The rest of the text describes the program goals well.